sdk/core/src/seekable_stream.rs

```diff
@@ -0,0 +1,97 @@
+use async_std::io::{Read, Seek, SeekFrom};
```

Comment:
I'm confused. We worked on removing a dependency on tokio, but now we have a dependency on async_std... I believe all these traits are from the futures crate no?

Reply:
Yes I've removed the entire crate from Cargo.toml along with the "file based" stream code.